### PR TITLE
chore(): make it a monorepo

### PR DESCRIPTION
**Pull request by boneskull**
in preparation for creating a tizen-websocket-talker lib...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
-  "name": "appium-tizen-tv-driver",
+  "name": "appium-tizen-tv-driver-monorepo",
+  "version": "0.3.0",
```
Comment (jlipps) [inline package.json:3]: do we want to set this to 0.0.1 or something that doesn't seem so sensible?

**Review by jlipps — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (boneskull) [reply package.json:3]: ah, right. this was a copy/paste job

Comment (boneskull) [reply package.json:3]: (this field is not actually used for or by anything)

**Comment by boneskull** [comment]
This also updates the driver to use the Appium peer dep and latest `@appium`-scoped packages.

**Comment by boneskull** [comment]
It does not remove gulp, but that looks pretty straightforward and can be done later.

**Pull request merged**